buildSrc/src/main/java/org/elasticsearch/gradle/precommit/ThirdPartyAuditTask.java

```diff
@@ -241,6 +249,10 @@ public void runThirdPartyAudit() throws IOException {
         }
 
         assertNoJarHell(jdkJarHellClasses);
+
+        // Mark successful third party audit check
+        getSuccessMarker().getParentFile().mkdirs();
+        Files.write(getSuccessMarker().toPath(), new byte[]{}, StandardOpenOption.CREATE);
```

Comment:
Wouldn't this fail on subsequent runs? I think we want the default options?

Reply:
I'm not sure, I think it just creates if it doesn't exist. I stole this from `PrecommitTask`. I think using the default options is probably the simpler thing to do though.